packages/fx-core/src/common/deps-checker/internal/nodeChecker.ts

```diff
@@ -179,7 +179,7 @@ export class SPFxNodeChecker extends NodeChecker {
   }
 
   protected async getSupportedVersions(): Promise<string[]> {
-    return ["12", "14", "16"];
+    return ["14", "16"];
```

Comment:
Also need to update env checker document?

https://github.com/OfficeDev/TeamsFx/blob/26d397a3713fab1d37364065874756ddea0399ca/docs/vscode-extension/envchecker-help.md?plain=1#L9
https://github.com/OfficeDev/TeamsFx/blob/26d397a3713fab1d37364065874756ddea0399ca/docs/vscode-extension/envchecker-help.md?plain=1#L60
https://github.com/OfficeDev/TeamsFx/blob/26d397a3713fab1d37364065874756ddea0399ca/docs/vscode-extension/envchecker-help.md?plain=1#L122

Reply:
Updated